iocore/net/P_UnixNet.h

```diff
@@ -243,32 +244,35 @@ class NetHandler : public Continuation, public EThread::LoopTailHandler
   uint32_t active_queue_size = 0;
 
   /// configuration settings for managing the active and keep-alive queues
-  struct Config {
-    uint32_t max_connections_in                 = 0;
-    uint32_t max_connections_active_in          = 0;
-    uint32_t inactive_threshold_in              = 0;
-    uint32_t transaction_no_activity_timeout_in = 0;
-    uint32_t keep_alive_no_activity_timeout_in  = 0;
-    uint32_t default_inactivity_timeout         = 0;
+  enum CONFIG {
+    max_connections_in,
+    max_connections_active_in,
+    inactive_threshold_in,
+    transaction_no_activity_timeout_in,
+    keep_alive_no_activity_timeout_in,
+    default_inactivity_timeout,
   };
+  static constexpr auto CONFIG_COUNT = static_cast<int>(CONFIG::default_inactivity_timeout) + 1;
+  /// Number of configuration variables / items.
+  using ConfigBlock = std::array<uint32_t, CONFIG_COUNT>;
```

Comment:
I thought 'using' violated Ape Law (or is this also just for me).

Reply:
That's for namespaces, not the equivalent of a `typedef`.